docs/TemplateManagementCLI.md

```diff
@@ -0,0 +1,75 @@
+Template Management is a tool to manage template files which is used in FHIR dotliquid converter engine. Template manager organizes template files in template image and allows pull and push operations. 
+
+Template image is a layer based structure similar to docker image and uses overlayfs concept to organize templates.
+
+For user's templates, we use two layers image structure to organize template collection: base layer and user layer (The user layer could be extended to multi-layers in the future if necessary). Base layer packs official published templates and user layer packs all modified templates from users. Each layer will be compressed into "*.tar.gz" file before pushing to ACR.
+# Using Template Management CLI
+
+The command-line tool can be used to pull and push a template collection through remote registry (Now we only support Azure Container Registry). Before pull & push operations, oras login is required.
+
```

Comment:
add this link to describe how to get the token
https://docs.microsoft.com/en-us/azure/container-registry/container-registry-authentication